### DISCUSSION TITLE
Will there be a mingw/gcc version?

### DISCUSSION THREAD

**Discussion by StarmanAkremis** [open-source-parsers/jsoncpp#1487] [General]
I would like to use this library in a microproject, so i use g++, i was wondering if i could download only the required stuff and keep things minimal
(Ok im going to be real now i wanted to use this in a mobile app called cxxdroid and i cant yse the ways that the documentation says to use the library)

**Comment by BillyDonahue** [comment]
You have all the sources.
Not sure what else you need?
